src/d3a/models/strategy/area_agents/two_sided_pay_as_bid_engine.py

```diff
@@ -225,6 +224,6 @@ def event_bid_split(self, *, market_id, original_bid, accepted_bid, residual_bid
                              f"{short_offer_bid_log_str(local_residual_bid)}")
 
     def _add_to_forward_bids(self, source_bid, target_bid):
-        bid_info = BidInfo(deepcopy(source_bid), deepcopy(target_bid))
+        bid_info = BidInfo(source_bid, target_bid)
```

Comment:
So I guess that the original objects of source_bid and target_bid stayed in memory and could not be deleted because the link was lost to them? Or is this something different?

Reply:
Indeed. I saw a huge amount of memory (about 50 MBs) of type `Bid` after using this tool https://pypi.org/project/memory-analyzer/ . Once I threw out the deepcopies, this was decreased by a lot.

Reply:
Awesome!